msprime/ancestry.py

```diff
@@ -851,6 +1154,9 @@ def __init__(
         ll_tables = _msprime.LightweightTableCollection(tables.sequence_length)
         ll_tables.fromdict(tables.asdict())
 
+        assert len(gene_conversion_map) == 1
```

Comment:
Is this assertion here because you want the user to specify a uniform gene conversion rate for now? It might be nice to raise a verbose error for the user about this

Reply:
opened #1212 to track.